src/components/Filters/RangeInput.tsx

```diff
@@ -102,6 +102,10 @@ export function RangeInput(props: RangeInputProps): JSX.Element | null {
   const [minRangeInput, setMinRangeInput] = useState<string>('');
   const [maxRangeInput, setMaxRangeInput] = useState<string>('');
   const staticFilters = useSearchState(state => state.filters.static);
+  const fieldValueFilters = useMemo(
+    () => getSelectableFieldValueFilters(staticFilters ?? []),
+    [staticFilters]
+  );
   const isDisabled = filters.some(filter => filter.selected && filter.fieldId === fieldId);
 
   const rangeFilter: NumberRangeFilter = useMemo(() => {
```

Comment:
Would it be possible to also put `kind: 'fieldValue'` here?

Reply:
updated!